components/style/restyle_hints.rs

```diff
@@ -694,6 +694,7 @@ impl DependencySet {
 #[cfg(all(test, feature = "servo"))]
 fn smoke_restyle_hints() {
     use cssparser::Parser;
+    use selectors::parser::ComplexSelector;
```

Comment:
The signature of `note_selector` changed, so this is probably not sufficient. :(